doc.go

```diff
@@ -433,7 +431,11 @@ type userContext struct {
 	// roles is a collection of roles the users holds
 	roles []string
 	// the access token itself
-	token jose.JWT
+	token *jwt.JSONWebToken
```

Comment:
If this `token` field isn't used anymore (with the introduction of `rawToken`), perhaps we can just remove the token field (or replace the `token` field with the `rawToken` string.